bluepyopt/ephys/protocols.py

```diff
@@ -245,12 +245,20 @@ def _reduce_method(meth):
             copyreg.pickle(types.MethodType, _reduce_method)
             import pebble
             from concurrent.futures import TimeoutError
+            import multiprocessing
+
+            # Default context for python>=3.8 on macos is spawn.
+            # Spwan context would reset neuron properties, such as dt.
+            multiprocessing_context = multiprocessing.get_context('fork')
```

Comment:
If fork does not reset the NEURON properties, why do we need this and to call set_neuron_variables?